ARDroneLib/VP_SDK/VP_Os/linux/intrin.h

```diff
@@ -9,22 +9,37 @@
 
 #if TARGET_CPU_X86 == 1
 
-static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
+/*static INLINE uint32_t _BitScanReverse(uint32_t* index, uint32_t mask)
```

Comment:
This is the part I am unsure about. I got warnings for this but the compilation finished on my laptop.